Makefile

```diff
@@ -27,6 +27,10 @@ endif
 build:
 	cargo build --release
 
+.PHONY: install
+install:
+	cargo install --path .
```

Comment:
ooh might be worth specifying `--locked` as well, otherwise it could pick up random versions of dependencies

Reply:
done! thanks